Loading/ru.lproj/InfoPlist.strings

```diff
@@ -1 +1 @@
-CFBundleName = "Загрузка";
CFBundleDisplayName = "Загрузка";
NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Программы\nВсе права защищены.";
\ No newline at end of file
+CFBundleName = "Loading";
CFBundleDisplayName = "Loading";
NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Apps\nВсе права защищены.";
```

Comment:
How come these are English? I actually meant to replace "Loading" in Localizable.strings with the translated versions, but forgot.